plutus-playground-server/app/PSGenerator.hs

```diff
@@ -290,22 +315,75 @@ multilineString :: Text -> Text -> Text
 multilineString name value =
     "\n\n" <> name <> " :: String\n" <> name <> " = \"\"\"" <> value <> "\"\"\""
 
+jsonExport :: ToJSON a => Text -> a -> Text
+jsonExport name value =
+    multilineString name (T.decodeUtf8 . BSL.toStrict $ JSON.encode value)
```

Comment:
is this pretty? if not then maybe it would be nice to be and also why multilineString? Oh, maybe it's because escaping is easier?

Reply:
Ah I see there is `encodePretty`. I think this would be nice as it's sometimes useful to read the test data and makes little to no performance or space difference.

Reply:
Great idea. 👍